x-pack/plugins/canvas/public/apps/workpad/workpad_app/workpad_telemetry.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+// @ts-ignore
+import { trackCanvasUiMetric } from '../../../lib/ui_metric';
+// @ts-ignore
+import { getElementCounts } from '../../../state/selectors/workpad';
+// @ts-ignore
+import { getArgs } from '../../../state/selectors/resolved_args';
+
+const WorkpadLoadedMetric = 'workpad-loaded';
+const WorkpadLoadedWithErrorsMetric = 'workpad-loaded-with-errors';
+
+export { WorkpadLoadedMetric, WorkpadLoadedWithErrorsMetric };
+
+const mapStateToProps = (state: any) => ({
+  telemetryElementCounts: getElementCounts(state),
+  telemetryResolvedArgs: getArgs(state),
+});
+
+interface ElementCounts {
```

Comment:
Please TSDoc these, and each property:
```
/**
 * Description
 */
```

This is the text that will appear in VSCode and elsewhere.